### ISSUE TITLE
Improve the Model

### ISSUE THREAD

**Issue by aarpit1010** [aarpit1010/Real-Time-Face-Mask-Detector#14]
Currently the model consists of a single dropout layer.

- In order to improve accuracy use dropout along with Batch Normalization after every Conv2D layer instead after last layer only.
- Use the number of kernels in powers of 2 (eg. 32, 64, 128, 256...) in place of the existing 100, 225...
- Also, be flexible and try for different values of dropout percentage.
- You can also increase the number of layers and change the epoch and batch size. 

Make sure these steps help in improving the accuracy of the model.
Any other creative changes in the model that can increase the accuracy are encouraged.

P.S. : pls. do follow the contributing guidelines while attempting the issue & sending PR...

**Comment by Tanish-16** [comment]
I want to work on this issue

**Comment by aarpit1010** [comment]
@Tanish-16  go ahead

**Comment by Tanish-16** [comment]
@aarpit1010 I tried changing the epochs size, number of layers and also tried by using batch normalization and dropout after every Conv2D layer, but for majority of the epochs there seems to be some or the other kind of error each time(for some of them the validation_accuracy comes out to be exactly 0.500, for some of them the validation_accuracy is very less compared to the training_accuracy and for some epochs, the validation_accuracy exceeds the training_accuracy). So , I am trying to change other few paramters as mentioned in the issue.

**Comment by aarpit1010** [comment]
ok ...
The main reason that I can see for all this could be due to the small dataset...
You could also do one other thing apart from the above changes and that is to implement transfer learning by using MobileNet or some other pre-trained model and modifying its last few layers...
But keep it as the last resort as it might be a separate issue that I would be creating in the coming days...
